src/index.js

```diff
@@ -0,0 +1,240 @@
+        tempLandscape = landscapes_map['60+']
+    } else if (temp >= 50) {
+        tempColorClass = 'fifty-degrees';
+        tempLandscape = landscapes_map['50+']
+    } else {
+        tempColorClass = 'forty-degrees';
+        tempLandscape = landscapes_map['40+']
+    }
+    return {
+        tempColorClass, 
+        tempLandscape
+    }
+}
+
+const setTempColor = (className) => {
+    state.tempBox.removeAttribute('class');
+    state.tempBox.classList.add(className);
+}
+
+const setTempLandscape = (landscapeType) => {
+    const landscape = document.getElementById('landscape');
+    landscape.textContent = landscapeType;
+}
+
+
+// FUNCTIONS FOR EACH ELEMENT'S EVENT HANDLER
+
+const registerIncreaseButton = () => {
+    const increaseButton = document.querySelector('#increase');
+    increaseButton.addEventListener('click', () => {
+        state.temp += 1
+        state.tempBox.textContent = `${state.temp} °F`;
+        styleColorAndLandscape();
+    });
+}
+
+const registerDecreaseButton = () => {
+    const decreaseButton = document.querySelector('#decrease');
+    decreaseButton.addEventListener('click', () => {
+        state.temp -= 1;
+        state.tempBox.textContent = `${state.temp} °F`;
+        styleColorAndLandscape();
+    })
+}
+
+const registerCityInput = () => {
+    const cityHeader = document.getElementById('display-city')
+    const cityInput= document.getElementById('city-name')
+    cityInput.addEventListener('input', () => {
+        cityHeader.innerHTML = cityInput.value
+    })
+}
+
+const registerCityResetButton = () => {
+    const resetButton = document.getElementById('reset');
+    const cityHeader = document.getElementById('display-city')
+    const cityInput= document.getElementById('city-name')
+    resetButton.addEventListener('click', () => {
+        cityHeader.innerHTML = 'Seattle';
+        cityInput.value = 'Seattle';
+    });
+}
+
+const registerSkySelect = () => {
+    const sky = document.getElementById('sky')
+    const skyMenu = document.getElementById('sky-select');
+    skyMenu.addEventListener('change', () => {
+        if (skyMenu.value == 'sunny') {
+            sky.textContent = state.sky_map.sunny;
+        } else if (skyMenu.value == 'cloudy') {
+            sky.textContent = state.sky_map.cloudy;
+        } else if (skyMenu.value == 'rainy') {
+            sky.textContent = state.sky_map.rainy;
+        } else {
+            sky.textContent = state.sky_map.snowy;
+        }
+    })
+};
+
+// REGISTER ALL EVENT HANDLERS
+const registerEventHandlers = () => {
+    registerIncreaseButton();
+    registerDecreaseButton();
+    registerCityInput();
+    registerCityResetButton();
+    registerSkySelect();
+}
+
+// RUN ON LOAD
+if (document.readyState !== 'loading') {
+    registerEventHandlers();
+} else {
+    document.addEventListener('DOMContentLoaded', registerEventHandlers);
+}
+
+
+
+
+
+
+
+
+
+// this function is responsible for changing two things in one function...
```

Comment:
Nice work refactoring and DRYing up your code! Remember to remove commented code for the final code.